src/terminal/adapter/FontBuffer.cpp

```diff
@@ -219,15 +219,15 @@ bool FontBuffer::FinalizeSixelData()
 
 gsl::span<const uint16_t> FontBuffer::GetBitPattern() const noexcept
 {
-    return { _buffer.data(), MAX_CHARS * _fullHeight };
+    return { _buffer.data(), gsl::narrow_cast<size_t>(MAX_CHARS * _fullHeight) };
 }
 
 til::size FontBuffer::GetCellSize() const noexcept
 {
-    return { gsl::narrow_cast<til::CoordType>(_fullWidth), gsl::narrow_cast<til::CoordType>(_fullHeight) };
+    return { _fullWidth, _fullHeight };
 }
 
-size_t FontBuffer::GetTextCenteringHint() const noexcept
+til::CoordType FontBuffer::GetTextCenteringHint() const noexcept

```

Comment:
This does not seem to be a coordinate

Reply:
The conversion of `FontBuffer` was the most difficult in my opinion.
It uses `size_t` for all numbers, but many of them are used for coordinated. I was very conflicted whether I should use `til::CoordType` for some of those or not.